### PR TITLE
NIP-34: add Status events, clarifications and improvements

### PR DESCRIPTION
**Pull request by DanConwayDev**
see commit messages for more details

### CODE REVIEW

File: 34.md [hunk 1]
```diff
@@ -17,48 +17,60 @@ Git repositories are hosted in Git-enabled servers, but their existence can be a
...
 
 Except `d`, all tags are optional.
 
 ## Patches
 
 Patches can be sent by anyone to any repository. Patches to a specific repository SHOULD be sent to the relays specified in that repository's announcement event's `"relays"` tag. Patch events SHOULD include an `a` tag pointing to that repository's announcement address.
 
+Patches in a patch set SHOULD include an nip10 `e` `reply` tag pointing to the to the previous patch. 
+
+The first patch revision in a patch revision SHOULD include a nip10 `e` `reply` to the oringal root patch.
```
Comment (fiatjaf) [inline 34.md:43]:
```suggestion
The first patch revision in a patch revision SHOULD include a nip10 `e` `reply` to the original root patch.
```

**Review by fiatjaf — APPROVED** [review]

File: 34.md (hunk 1 above)
Comment (DanConwayDev) [reply 34.md]: I made this updated via an amended commit

File: 34.md [hunk 2]
```diff
@@ -17,48 +17,60 @@ Git repositories are hosted in Git-enabled servers, but their existence can be a
...
-The tags `web`, `clone`, `relays` can have multiple values.
+The tags `web`, `clone`, `relays`, `maintainers` can have multiple values.
 
 Except `d`, all tags are optional.
 
 ## Patches
 
 Patches can be sent by anyone to any repository. Patches to a specific repository SHOULD be sent to the relays specified in that repository's announcement event's `"relays"` tag. Patch events SHOULD include an `a` tag pointing to that repository's announcement address.
 
+Patches in a patch set SHOULD include an NIP-10 `e` `reply` tag pointing to the to the previous patch.
```
Comment (dluvian) [inline 34.md:41]:
```suggestion
Patches in a patch set SHOULD include a NIP-10 `e` `reply` tag pointing to the previous patch. 
```

**Comment by vitorpamplona** [comment]
Nice!

**Comment by DanConwayDev** [comment]
unfortunately we have to filter client side as every root patch / issue begins its life in the Open state.

> Root Patches and Issues have a Status that defaults to 'Open'

This is how I'm doing it:
1. search for all root patches / issues and Status events associated with a repository (via `unique-commit` tag or `a` tag)
2. mop up by requesting status events replying to root patches / issues where the `unique-commit` tag or `a` tag is omitted
3. filter by Open on the client side: filter out root patches / issues, that have a most recent Status event other than Open

File: 34.md (hunk 2 above)
Comment (DanConwayDev) [reply 34.md]: I made this update via an amended commit

**Pull request merged**